### ISSUE TITLE
Produkt tymczasowy jest przypisany do innej oferty

### ISSUE THREAD

**Issue by bartexs6** [allegro/allegro-api#9599]
Witam,

Od kilku dni na niektórych wystawianych produktach wyskakuję błąd "Podany produkt tymczasowy jest przypisany do innej oferty", mimo, że produkty są oznaczone jako nowe.

Trace-id=[1730aa72a143fcdd]
Trace-id=[e1c290f98d6590ff]

**Comment by AureliuszBrussy** [comment]
Próbujesz wystawić ofertę korzystając z produktu tymczasowego, który utworzony jest tylko na potrzeby konkretnej oferty. Więcej o produktach tymczasowych przeczytasz w [naszym newsie](https://developer.allegro.pl/news/w-wybranych-kategoriach-bedziemy-tworzyc-produkty-tymczasowe-nn9Y12dbqHM). Nie możesz wykorzystać tego produktu przy wystawianiu kolejnej oferty.

**Comment by bartexs6** [comment]
Czemu więc korzystam z produktu tymczasowego skoro nie ma on statusu innego niż nowy?

"w których sprzedawca dla parametru “Stan” wybrał wartość inną niż “Nowy”"

**Comment by AureliuszBrussy** [comment]
Niestety, ale nie rozumiem pytania. Produkt tymczasowy tworzony jest na potrzeby oferty, której stan jest inny niż "Nowy", oraz tylko i wyłącznie na potrzeby tej jednej konkretnej oferty. Po odpytaniu o szczegóły tego produktu (GET /sale/products/921f0f30-c6dc-4f23-aae2-5481e5a165d4) znajdziesz w polu `offerRequirements` informację o tym, na potrzeby której oferty ten produkt został utworzony. Nie możesz go użyć podczas próby wystawienia innej oferty.

**Comment by bartexs6** [comment]
Rozumiem, lecz produkty, które są wystawiane mają stan produktu "Nowy".

Czy coś zmieniło się w okolicach 1 sierpnia w sprawie produktów tymczasowych?

**Comment by AureliuszBrussy** [comment]
Nie, nic się nie zmieniło. W swoim requeście wskazujesz konkretny produkt, który jest tymczasowy. Nie ma znaczenia, jaki stan wskażesz. Ten konkretny produkt został stworzony na potrzeby jednej, konkretnej oferty.
